### DISCUSSION TITLE
Init Config with Transformers

### DISCUSSION THREAD

**Discussion by algo-sroy** [explosion/spaCy#8431] [Help: Coding & Implementations]
I am trying to fill the config file with Transformers parameters using the command given below. Since it populates the transformers only when GPU support is activated i am using -G for that. My machine currently does not possess a GPU.

    python -m spacy init config ./config_test2.cfg --pipeline ner -G -F

Error : jinja2.exceptions.UndefinedError: 'None' has no attribute 'efficiency'
Is there any alternate way to get the Transformers parameters using CLI commands?

**Comment by adrianeboyd** [comment]
Hi, can you check that you're using the latest version of spacy? I think I remember a bug fix in the template used for the recommendations, but it was quite a while ago. The current version should be v3.0.6.
